docs/sql/reference/insert.txt

```diff
@@ -35,6 +40,26 @@ not be filled.
 If the expression for any column is not of the correct data type,
 automatic type conversion will be attempted.
 
+On Duplicate Key Update
+-----------------------
+
+If ``ON DUPLICATE KEY UPDATE`` is specified and a row is inserted that
+would cause a duplicate PRIMARY KEY value, an update of the existing
+row is performed.
+
+::
+
+      ON DUPLICATE KEY UPDATE
+      { column_ident = expression |
+        ( column_ident [, ...] ) = ( expression [,...] )
+      } [, ...]
+
+Within expressions in the ``UPDATE`` clause you can use the
+``VALUES(column_ident)`` function to refer to column values
+from the INSERT statement.
+So ``VALUES(column_ident)`` in the ``UPDATE`` clause referes to the
+value of the column_ident that would be inserted if no duplicate-key
+conflict occured.
```

Comment:
explain that this usually makes sense if the insert statement contains multiple values so that the values for the current row can be specified